jsonschema-hyperschema.xml

```diff
@@ -116,12 +121,20 @@
                 The terms "schema" and "instance" are to be interpreted as defined in the
                 <xref target="json-schema">JSON Schema core specification</xref>.
             </t>
+
+            <t>
+                This specification uses the terms "link context" and "link target" as
+                defined in <xref target="I-D.nottingham-rfc5988bis"/>.  These terms respectively
+                correspond with "Context IRI" and "Target IRI" as used in
+                <xref target="RFC5988">RFC 5988</xref>.
+                Although defined as IRIs, in common scenarios they are also URIs.
```

Comment:
It seems that RFC5988bis has been approved as a "Proposed Standard" (https://datatracker.ietf.org/doc/draft-nottingham-rfc5988bis/writeup/). So should we keep references to the old RFC?